net/ampr-ripd/patches/001-ampr-ripd.c.patch

```diff
@@ -0,0 +1,12 @@
+--- ampr-ripd-2.4/ampr-ripd.c	2017-08-14 21:43:56.000000000 -0400
```

Comment:
Git header is missing. Is this included in upstream? If not, you should describe what it does and then submit it to upstream.

Reply:
Please elaborate.  Not sure what action to take.

Reply:
The added lines are necessary for the code to compile.  It is not clear why they are not in the original source code.

Reply:
While using your words and sorry for that, it is not clear, why we should have this patch in our repository, when it is not possible to compile it from the source code. In that case, you should report it to upstream, fill bug report, try to convince them to include it (if possible) and have it included in the next version. We should not carry the patch and keep it in our repository and take care of it. As I said earlier:

This patch is missing, who is the author, commit subject, commit description, etc.